src/data/documentation-links.yaml

```diff
@@ -22,7 +22,7 @@
       to: /docs/fundamentals/sync-modes
     - id: Account management
       to: /docs/fundamentals/account-management
-    - id: Backup restore
+    - id: 'Backup & Restore'
```

Comment:
We don't appear to be perfectly consistent with this, but most of the titles here are using sentence casing... I think it would help tidy up if we picked between sentence case or title case and make all of these consistent. 

Just noting this here, don't think it needs to block this PR.

Reply:
Agree we should be consistent to avoid issues

Reply:
```suggestion
    - id: 'Backup & restore'
```